azext_iot/central/_help.py

```diff
@@ -107,6 +107,20 @@ def _load_central_devices_help():
             --device-id {deviceid}
     """
 
+    helps[
+        "iot central app device get-credentials"
```

Comment:
For commands that provide say 'CRUDL' for a resource, as much as possible we want to stick with standard CLI verbs i.e. Create/Show/Update/Delete/List...for one off's there is more flexibility but why not change this command to `show-credentials`?

All that being said have a related comment on this command in a different file.